android/src/main/java/io/invertase/googlemobileads/ReactNativeAppModule.java

```diff
@@ -55,6 +57,17 @@ public void initializeApp(ReadableMap options, ReadableMap appConfig, Promise pr
     promise.resolve(options);
   }
 
+  @ReactMethod(isBlockingSynchronousMethod = true)
```

Comment:
I have gotten a lot of negative feedback from module users historically when I've used this annotation - it has the side effect of making it so that the JSI debugging interface won't work, as you cannot use synchronous methods and the JSI. Is this strictly necessary, with no way around it? It would be the first instance of it in this module and going from 0 to 1 is a big step...

Reply:
If we make asynchronous  - we will face another inconvenience - we will need somehow to get this value before showing banner or there still be fast content offset

Reply:
Hmm - if it is just order of operations then I'd rather it be async with good documentation on how to call this first and/or make a best guess first then call this etc to block the space out, rather than forced sync and breaking JSI usage - all I can tell you from experience is that you really don't want to do that if you can avoid it. You (I? we? :laughing: ) will get a *lot* of complaints and invariably you'll have a sync and async version which is just a pain

Reply:
We also already have a similar static method: https://github.com/invertase/react-native-google-mobile-ads/blob/main/android/src/main/java/io/invertase/googlemobileads/ReactNativeGoogleMobileAdsCommon.java#L43